pkg/uhttp/wrapper.go

```
@@ -403,6 +404,17 @@
 		}
 	}
 
+	// Log response headers directly for certain errors
+	if resp != nil {
```

golangci-lint (1.64.8)

SA5011(related information): this check suggests that the pointer can be nil

(staticcheck)